components/table/README.md

```diff
@@ -91,23 +137,22 @@ When tabular data can be sorted, the `<d2l-table-col-sort-button>` can be used t
 </table>
 ```
 
-**Properties:**
+### Properties
 
 | Property | Type | Description |
 |--|--|--|
```

Comment:
Might require testing again but I think if we want table styles to apply properly to inline markdown tables they need to have 3 dashes to specify the header line 
```suggestion
|---|---|---|
```